src/autosuggest/__integ__/async-autosuggest.test.ts

```diff
@@ -138,7 +138,7 @@ describe.each([true, false])('Async autosuggest scrolling (with virtualScrolling
     'the bottom of the dropdown lines up with the highlighted option, when going down the list',
     setup({ virtualScrolling }, async page => {
       await page.respondWith(0, true);
-      await page.keys(['ArrowDown', 'ArrowDown', 'ArrowDown']);
+      await page.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown']);
```

Comment:
What is happening here and why is unrelated switcher change causes a possible breaking change here?

Reply:
On smaller screen sizes the header now takes two lines instead of one - that caused some tests that depend on the page height to fail.

Reply:
I was going to suggest removing the header from screenshot tests as well. As far as next steps, who is the appropriate owner of these changes?

Reply:
Let's sync in our team chat and try to see how much we can do within this project

Reply:
For visibility I am going to open a separate PR that handles the demo pages controls in a way that prevents wrapping. Then, we could rebase and merge this PR.